src/components/DonutChart/DonutChart.js

```diff
@@ -0,0 +1,51 @@
+import React, { useRef, useEffect } from "react";
+import * as d3 from "d3";
+
+const DonutChart = ({ chartData }) => {
+  const ref = useRef();
+  const width = 220;
+  const height = 220;
+  const margin = 40;
+
+  useEffect(() => {
+    const radius = Math.min(width, height) / 2 - margin;
+
+    const svg = d3
+      .select(ref.current)
+      .attr("width", width)
+      .attr("height", height)
+      .append("g")
+      .attr("transform", "translate(" + width / 2 + "," + height / 2 + ")");
+
+    const data = chartData;
+
+    const color = d3
+      .scaleOrdinal()
+      .domain(data)
+      .range(["#c7162b", "#f99b11", "#cdcdcd"]);
```

Comment:
Just an idea but if we assign these colors to CSS variables, we can then access them via JS

https://davidwalsh.name/css-variables-javascript